pyiron/dft/waves/electronic.py

```diff
@@ -633,10 +670,10 @@ def plot_fermi_dirac(self):
             import matplotlib.pylab as plt
         except ModuleNotFoundError:
             import matplotlib.pyplot as plt
-        arg = np.argsort(self.eigenvalues)
-        plt.plot(
-            self.eigenvalues[arg], self.occupancies[arg], linewidth=2.0, color="blue"
-        )
+        for spin, eigenvalues in enumerate(self.eigenvalues):
+            arg = np.argsort(eigenvalues)
+            plt.plot(eigenvalues[arg], self.occupancies[spin][arg], linewidth=2.0, label="spin:{}".format(spin))
+        plt.legend()
         plt.axvline(self.efermi, linewidth=2.0, linestyle="dashed", color="black")
         plt.xlabel("Energies (eV)")
```

Comment:
Here maybe rather "eigen value"? I mean it's true that it's related to the potential energy but Oxford dictionary states *the property of matter and radiation which is manifest as a capacity to perform work* and since it is not a quantity for the system to perform work, I'd find it more appropriate to call it "eigen value". But if you prefer energy I'm also fine with it

Reply:
Did you mean this for the axis label?

Reply:
Yes

Reply:
done